sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -462,6 +462,7 @@ def create_item(
         pre_trigger_include=None,  # type: Optional[str]
         post_trigger_include=None,  # type: Optional[str]
         indexing_directive=None,  # type: Optional[Any]
+        disable_automatic_id_generation=None, #type: Optional[bool]
```

Comment:
Could this be moved into `kwargs`?
(The `:param` label in the docstring would move to `:keyword` to match.

Reply:
Maybe it's true to make it easier for those that don't have an id to use.

Reply:
@annatisch I see no reason why not, however all of the other similar args follow this pattern

Reply:
Thanks @zfoster - we already have more positional optional parameters in the method signatures than we would generally advise, as making these parameters keyword only gives us more options for non-breaking changes in the future.
So going forward we should generally add new parameters to `kwargs` unless there's a very compelling reason not to.

Reply:
@annatisch do you happen to know why this is the case in python?

https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/test/test_crud.py#L698

Reply:
or by chance know who _would_ know? :)

Reply:
Spark Connector creates an id by default. Maybe it's for consistency.

Reply:
It's doing the opposite here I think? We have hardcoded to DISABLE ID generation

Reply:
@zfoster I think that comment is just wrong. We should match whatever the JS SDK does here.